Assets/Scripts/Enums/Enums.cs

```diff
@@ -7,7 +7,18 @@ namespace Game.Enums
     public enum SceneName
     {
         Scene_Main,
-        Scene_Dialogue
+        Scene_Dialogue,
+        Scene_PortalTest
+    }
+
+    public enum DestinationIdentifier
+    {
```

Comment:
Can you use more descriptive identifiers here?

Reply:
Will do.